README.md

```diff
@@ -109,8 +109,42 @@ export ETH_SIGNING_KEY=<Your ETH SIGNING KEY>
 pigeon start
 ```
 
-- Remember to run pigeon as a systemd Service! If you have a good systemd service implementation for Ubuntu, please make a PR on this README and we will add it.
-
+#### Using systemd service
+
+```bash
```

Comment:
````
```shell
````

Reply:
you mean `shell script` ? I don't readme  highlight just with shell